jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -41,8 +44,9 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
         raise TypeError("SpectrumCollection detected."
                         " Please provide a Spectrum1D or SpectrumList")
     elif isinstance(data, Spectrum1D):
-        data = [data]
+        # should we make a copy if Spectrum1D is passed in to avoid modifying it when uncerts are reset to None?
```

Comment:
If you pass in a Spectrum1D with uncertainty=[nan, nan, nan...], the input will be modified without a copy, with uncertainty now being None. What do you think about that, is it worth making a copy to avoid modifying input? Im not sure how it works elsewhere in the code if any modifications are made to the input spectrum